biscuit/src/Auth/Biscuit/Datalog/Parser.hs

```diff
@@ -338,38 +340,38 @@ checkParser = do
   cKind <- l $ choice [ One <$ chunk "check if"
                       , All <$ chunk "check all"
                       ]
-  cQueries <- queryParser
+  cQueries <- queryParser <?> "datalog query"
```

Comment:
Shouldn't it be done in the `queryParser` function directly?

Reply:
right, that would be better. I try to keep annotations at use site to account for context-dependent help, but in that case that's not needed.